notanorm/base.py

```diff
@@ -315,6 +325,11 @@ def __resolve_on_field(self, k, tab):
         return k
 
 
+class Where:
+    def __init__(self, ops: List[Dict[str, Union[Op, SubQ]]]):
```

Comment:
Why a list of dicts? Seems fine, just not sure why it's done this way.

Reply:
because the underlying code already supported "dicts" as the mechanism for specifying key/value.